jobs/JEVS_MESOSCALE_PLOTS

```diff
@@ -102,7 +102,7 @@ fi
 # Define COMIN/COMOUT variables:
 ####################################
 export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
-export COMINapcp24mean=${COMINspatial:-$(compath.py  $envir/com/$NET/$evs_ver)}/stats/$COMPONENT
+export COMINapcp24mean=${COMINapcp24mean:-$(compath.py  $envir/com/$NET/$evs_ver)}/stats/$COMPONENT
```

Comment:
@ShelleyMelchior-NOAA @BinbinZhou-NOAA Will this change to "export COMINapcp24mean=${COMINapcp24mean" hurt anything from Roshan's previous mesoscale PR that was already merged?

Reply:
The line 105 in Binbin's PR is correct. So it will correct develop, which is good. Roshan's mesoscale precip plots driver does not specify COMINspatial, and COMINapcp24mean is set in the driver to override the J-job setting. I think this is square.